arelle/plugin/inlineXbrlDocumentSet.py

```diff
@@ -76,7 +76,12 @@
 - For EDGAR style encoding of non-ASCII characters, use the `--encodeSavedXmlChars` argument.
 - Extracted XML instance is saved to the same directory as the IXDS with the suffix `_extracted.xbrl`.
 """
-from arelle import FileSource, ModelXbrl, ValidateXbrlDimensions, XbrlConst
+from __future__ import annotations
```

Comment:
Is this needed?

Reply:
Yes, gradually added some type hints with cutting edge '|' notation.